### ISSUE TITLE
Tweet list long press action

### ISSUE THREAD

**Issue by M4rtinK** [SfietKonstantin/harbour-twablet#73]
The long press action is already quite established on Sailfish OS and it could be used to open a context menu for individual tweets in the tweet list view. This menu could be used to trigger often used actions without visiting the tweet detail screen. 

Possible context menu content:
- favorite
- retweet
- reply

**Comment by SfietKonstantin** [comment]
Agreed. Let's do this for 0.8 though, as 0.7 is already packed with features.

**Comment by M4rtinK** [comment]
Sure. :)
